templates/lobbys/lobby.html

```diff
@@ -177,10 +226,58 @@ <h2 v-else>Lobby [[lobbyInfo.lobby_id]]: </h2>
         <span id="custom-tooltip" ref="shareTooltip">Copied invite to clipboard!</span>
     </div>
 
+    
 
     <template v-if="lobbyInfo.user.owner">
+
         <hr>
-        <h4>Manage: Add a prompt</h4>
+
+        <h3>Lobby settings</h4>
+
+        <details>
+            <summary>List of players</summary>
+            <div class="card">
+                <div class="card-body">
+                    <table class="table table-hover">
+                        <thead>
+                            <th scope="col">Display name</th>
+                            <th scope="col">User type</th>
+                            <th scope="col"></th>
+                        </thead>
+                        <tbody>
+                            <tr v-for="user in users" v-cloak>
+                                <td>[[user.username]]</td>
+                                <td>Registered User</td>
+                                <td>
+                                    <div class="dropdown">
+                                        <button class="btn btn-secondary dropdown-toggle" 
+                                                type="button" 
+                                                data-toggle="dropdown" 
+                                                aria-haspopup="true" 
+                                                aria-expanded="true"
+                                                @click="toggleDropdown(String(user.user_id)+'dropdown')">
+                                            <small>Options</small>
+                                        </button>
+                                        <div class="dropdown-menu" 
+                                            v-bind:id="String(user.user_id)+'dropdown'">
+                                            <button v-if="!user.owner" class="btn btn-sm" @click="makeHost(user.user_id, user.username)">Transfer Ownership</button>
+                                            <button class="btn btn-sm disabled" v-else>Currently host</button>
```

Comment:
(personal preference)
Don't really think the "Currently host" is needed since it's implied, so I would probably just delete this line. I was kind of confused looking at the example image.